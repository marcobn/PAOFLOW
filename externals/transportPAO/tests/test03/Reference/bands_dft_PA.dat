    0.000000000  -16.358100891
    0.020000000  -16.351400375
    0.040000000  -16.331800461
    0.060000000  -16.298801422
    0.080000000  -16.252300262
    0.100000000  -16.193000793
    0.120000000  -16.121101379
    0.140000000  -16.035301208
    0.160000000  -15.936800003
    0.180000000  -15.826101303
    0.200000000  -15.702199936
    0.220000000  -15.564701080
    0.240000000  -15.415000916
    0.260000000  -15.253000259
    0.280000000  -15.078599930
    0.300000000  -14.891500473
    0.320000000  -14.693099976
    0.340000000  -14.482700348
    0.360000000  -14.261800766
    0.380000000  -14.031600952
    0.400000000  -13.790500641
    0.420000000  -13.542301178
    0.440000000  -13.291500092
    0.460000000  -13.045200348
    0.480000000  -12.833400726
    0.500000000  -12.737100601
 
    0.000000000   -9.792499542
    0.020000000   -9.785999298
    0.040000000   -9.767499924
    0.060000000   -9.736799240
    0.080000000   -9.693500519
    0.100000000   -9.639099121
    0.120000000   -9.573499680
    0.140000000   -9.496099472
    0.160000000   -9.409299850
    0.180000000   -9.314899445
    0.200000000   -9.213500977
    0.220000000   -9.109899521
    0.240000000   -9.013500214
    0.260000000   -8.944400787
    0.280000000   -8.945999146
    0.300000000   -9.072000504
    0.320000000   -9.315500259
    0.340000000   -9.621999741
    0.360000000   -9.951799393
    0.380000000  -10.288799286
    0.400000000  -10.624599457
    0.420000000  -10.954799652
    0.440000000  -11.270299911
    0.460000000  -11.564199448
    0.480000000  -11.804599762
    0.500000000  -11.911399841
 
    0.000000000   -4.741500378
    0.020000000   -4.758900166
    0.040000000   -4.816299915
    0.060000000   -4.915999889
    0.080000000   -5.060200214
    0.100000000   -5.252500057
    0.120000000   -5.490100384
    0.140000000   -5.760799885
    0.160000000   -6.059000492
    0.180000000   -6.381200314
    0.200000000   -6.714000225
    0.220000000   -7.048900127
    0.240000000   -7.376300335
    0.260000000   -7.672399998
    0.280000000   -7.890599728
    0.300000000   -7.972799778
    0.320000000   -7.927899837
    0.340000000   -7.810100079
    0.360000000   -7.655000210
    0.380000000   -7.479799747
    0.400000000   -7.290000439
    0.420000000   -7.092500210
    0.440000000   -6.889700413
    0.460000000   -6.685699940
    0.480000000   -6.490100384
    0.500000000   -6.368600368
 
    0.000000000   -4.735300541
    0.020000000   -4.726899624
    0.040000000   -4.701300144
    0.060000000   -4.657300472
    0.080000000   -4.594999790
    0.100000000   -4.516200542
    0.120000000   -4.420799732
    0.140000000   -4.307199955
    0.160000000   -4.176700115
    0.180000000   -4.126400471
    0.200000000   -4.171900272
    0.220000000   -4.224299908
    0.240000000   -4.286900043
    0.260000000   -4.361000538
    0.280000000   -4.447900295
    0.300000000   -4.547299862
    0.320000000   -4.660299778
    0.340000000   -4.789700031
    0.360000000   -4.932799816
    0.380000000   -5.090300083
    0.400000000   -5.258400440
    0.420000000   -5.438299656
    0.440000000   -5.625899792
    0.460000000   -5.819500446
    0.480000000   -6.008900166
    0.500000000   -6.131200314
 
    0.000000000   -3.793099880
    0.020000000   -3.803200245
    0.040000000   -3.833400249
    0.060000000   -3.874900341
    0.080000000   -3.919900417
    0.100000000   -3.965099812
    0.120000000   -4.007400036
    0.140000000   -4.045699596
    0.160000000   -4.083799839
    0.180000000   -4.030499935
    0.200000000   -3.866600513
    0.220000000   -3.685400009
    0.240000000   -3.487100124
    0.260000000   -3.274000168
    0.280000000   -3.044400215
    0.300000000   -2.799300194
    0.320000000   -2.539100170
    0.340000000   -2.264299870
    0.360000000   -1.975800037
    0.380000000   -1.675199986
    0.400000000   -1.361700058
    0.420000000   -1.040699959
    0.440000000   -0.716500282
    0.460000000   -0.400500298
    0.480000000   -0.126999855
    0.500000000    0.000000000
 
    0.000000000    3.960500002
    0.020000000    3.970499992
    0.040000000    3.999799967
    0.060000000    4.048500061
    0.080000000    4.116199970
    0.100000000    4.202700138
    0.120000000    4.307499886
    0.140000000    4.430500031
    0.160000000    4.570199966
    0.180000000    4.721899986
    0.200000000    4.855999947
    0.220000000    4.833899975
    0.240000000    4.763799667
    0.260000000    4.637000084
    0.280000000    4.452600002
    0.300000000    4.215000153
    0.320000000    3.931599855
    0.340000000    3.611099958
    0.360000000    3.264800072
    0.380000000    2.898200035
    0.400000000    2.523499966
    0.420000000    2.145900011
    0.440000000    1.772500038
    0.460000000    1.419800043
    0.480000000    1.123199940
    0.500000000    0.987299919
 
    0.000000000    4.096899986
    0.020000000    4.105400085
    0.040000000    4.131000042
    0.060000000    4.173500061
    0.080000000    4.232599735
    0.100000000    4.307600021
    0.120000000    4.398099899
    0.140000000    4.503499985
    0.160000000    4.623099804
    0.180000000    4.759500027
    0.200000000    4.872399807
    0.220000000    5.019499779
    0.240000000    5.167099953
    0.260000000    5.314300060
    0.280000000    5.460299969
    0.300000000    5.605500221
    0.320000000    5.752299786
    0.340000000    5.904099941
    0.360000000    6.065299988
    0.380000000    6.240099907
    0.400000000    6.431799889
    0.420000000    6.642999649
    0.440000000    6.875100136
    0.460000000    7.127699852
    0.480000000    7.396299839
    0.500000000    7.602399826
 
    0.000000000    4.489399910
    0.020000000    4.496200085
    0.040000000    4.515399933
    0.060000000    4.546499729
    0.080000000    4.588500023
    0.100000000    4.638899803
    0.120000000    4.694699764
    0.140000000    4.752399921
    0.160000000    4.804599762
    0.180000000    4.842199802
    0.200000000    4.924200058
    0.220000000    5.116399765
    0.240000000    5.329299927
    0.260000000    5.560599804
    0.280000000    5.809899807
    0.300000000    6.077099800
    0.320000000    6.361499786
    0.340000000    6.663000107
    0.360000000    6.981200218
    0.380000000    7.315400124
    0.400000000    7.665499687
    0.420000000    8.028800011
    0.440000000    8.393400192
    0.460000000    8.375699997
    0.480000000    8.056799889
    0.500000000    7.831299782
 
    0.000000000    4.736499786
    0.020000000    4.746299744
    0.040000000    4.775700092
    0.060000000    4.824800014
    0.080000000    4.893499851
    0.100000000    4.981800079
    0.120000000    5.089699745
    0.140000000    5.217199802
    0.160000000    5.364399910
    0.180000000    5.531000137
    0.200000000    5.717199802
    0.220000000    5.900799751
    0.240000000    6.100699902
    0.260000000    6.317999840
    0.280000000    6.553699970
    0.300000000    6.809499741
    0.320000000    7.086500168
    0.340000000    7.385399818
    0.360000000    7.706500053
    0.380000000    8.049799919
    0.400000000    8.414499283
    0.420000000    8.798399925
    0.440000000    8.769399643
    0.460000000    8.829099655
    0.480000000    9.219200134
    0.500000000    9.587699890
 
    0.000000000    4.811100006
    0.020000000    4.820499897
    0.040000000    4.848599911
    0.060000000    4.895400047
    0.080000000    4.960599899
    0.100000000    5.043999672
    0.120000000    5.144999981
    0.140000000    5.263299942
    0.160000000    5.398399830
    0.180000000    5.549699783
    0.200000000    5.717199802
    0.220000000    5.922999859
    0.240000000    6.148399830
    0.260000000    6.393199921
    0.280000000    6.657599926
    0.300000000    6.941400051
    0.320000000    7.244699955
    0.340000000    7.567299843
    0.360000000    7.909299850
    0.380000000    8.270500183
    0.400000000    8.650899887
    0.420000000    9.050300598
    0.440000000    9.212599754
    0.460000000    9.637800217
    0.480000000    9.987499237
    0.500000000    9.657199860
 
    0.000000000    4.873700142
    0.020000000    4.883199692
    0.040000000    4.911600113
    0.060000000    4.959000111
    0.080000000    5.025499821
    0.100000000    5.111299992
    0.120000000    5.216400146
    0.140000000    5.341199875
    0.160000000    5.485699654
    0.180000000    5.650000095
    0.200000000    5.834199905
    0.220000000    6.038199902
    0.240000000    6.262100220
    0.260000000    6.505700111
    0.280000000    6.769000053
    0.300000000    7.051799774
    0.320000000    7.354199886
    0.340000000    7.675899982
    0.360000000    8.016799927
    0.380000000    8.376600266
    0.400000000    8.754799843
    0.420000000    9.120300293
    0.440000000    9.468199730
    0.460000000    9.904199600
    0.480000000   10.095299721
    0.500000000   10.522600174
 
    0.000000000    5.269499779
    0.020000000    5.279500008
    0.040000000    5.309199810
    0.060000000    5.358699799
    0.080000000    5.427999973
    0.100000000    5.516799927
    0.120000000    5.625299931
    0.140000000    5.753299713
    0.160000000    5.900799751
    0.180000000    6.067699909
    0.200000000    6.254199982
    0.220000000    6.460100174
    0.240000000    6.685600281
    0.260000000    6.930399895
    0.280000000    7.194600105
    0.300000000    7.477999687
    0.320000000    7.766699791
    0.340000000    8.058599472
    0.360000000    8.374799728
    0.380000000    8.714599609
    0.400000000    9.077300072
    0.420000000    9.151100159
    0.440000000    9.559999466
    0.460000000    9.960599899
    0.480000000   10.356699944
    0.500000000   10.578599930
 
    0.000000000    5.420300007
    0.020000000    5.430099964
    0.040000000    5.459599972
    0.060000000    5.508699894
    0.080000000    5.577499866
    0.100000000    5.665900230
    0.120000000    5.774000168
    0.140000000    5.901700020
    0.160000000    6.048899651
    0.180000000    6.215700150
    0.200000000    6.401899815
    0.220000000    6.606900215
    0.240000000    6.826700211
    0.260000000    7.038100243
    0.280000000    7.257400036
    0.300000000    7.499599934
    0.320000000    7.778900146
    0.340000000    8.080299377
    0.360000000    8.395199776
    0.380000000    8.733400345
    0.400000000    9.092700005
    0.420000000    9.462100029
    0.440000000    9.868000031
    0.460000000   10.293699265
    0.480000000   10.529600143
    0.500000000   10.807000160
 
    0.000000000    5.584000111
    0.020000000    5.593599796
    0.040000000    5.622200012
    0.060000000    5.669899940
    0.080000000    5.736499786
    0.100000000    5.821700096
    0.120000000    5.925299644
    0.140000000    6.046700001
    0.160000000    6.184700012
    0.180000000    6.337900162
    0.200000000    6.504099846
    0.220000000    6.678199768
    0.240000000    6.857999802
    0.260000000    7.069499969
    0.280000000    7.288399696
    0.300000000    7.529299736
    0.320000000    7.796199799
    0.340000000    8.106699944
    0.360000000    8.447500229
    0.380000000    8.808000565
    0.400000000    9.184499741
    0.420000000    9.484600067
    0.440000000    9.888000488
    0.460000000   10.313699722
    0.480000000   10.732799530
    0.500000000   10.821499825
 
    0.000000000    5.750299931
    0.020000000    5.758699894
    0.040000000    5.783100128
    0.060000000    5.823699951
    0.080000000    5.880000114
    0.100000000    5.951600075
    0.120000000    6.037799835
    0.140000000    6.138400078
    0.160000000    6.252399921
    0.180000000    6.379600048
    0.200000000    6.520899773
    0.220000000    6.680999756
    0.240000000    6.868599892
    0.260000000    7.086099625
    0.280000000    7.348799706
    0.300000000    7.632800102
    0.320000000    7.936800003
    0.340000000    8.260399818
    0.360000000    8.603599548
    0.380000000    8.966300011
    0.400000000    9.348400116
    0.420000000    9.594200134
    0.440000000   10.010000229
    0.460000000   10.444799423
    0.480000000   10.759599686
    0.500000000   10.936399460
 
