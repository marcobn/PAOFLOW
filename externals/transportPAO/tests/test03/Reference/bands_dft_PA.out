
     Program PWSCF v.5.0.1 starts on 28Feb2013 at 10:20:39 

     This program is part of the open-source Quantum ESPRESSO suite
     for quantum simulation of materials; please cite
         "P. Giannozzi et al., J. Phys.:Condens. Matter 21 395502 (2009);
          URL http://www.quantum-espresso.org", 
     in publications or presentations arising from this work. More details at
     http://www.quantum-espresso.org/quote.php

     Parallel version (MPI), running on     6 processors
     R & G space division:  proc/pool =    6

     Current dimensions of program PWSCF are:
     Max number of different atomic species (ntypx) = 10
     Max number of k-points (npk) =  40000
     Max angular momentum in pseudopotentials (lmaxx) =  3
     Reading input from /home/ferretti/want/tests/test16/bands_dft_PA.in
               file H.pz-vbc.UPF: wavefunction(s)  1S renormalized

     Atomic positions and unit cell read from directory:
     ./SCRATCH/poly_pa.save/
 

     Subspace diagonalization in iterative solution of the eigenvalue problem:
     a serial algorithm will be used

 
     Parallelization info
     --------------------
     sticks:   dense  smooth     PW     G-vecs:    dense   smooth      PW
     Min        1907    1907    584                23948    23948    4049
     Max        1909    1909    586                23949    23949    4050
     Sum       11449   11449   3513               143691   143691   24299
 


     bravais-lattice index     =            8
     lattice parameter (alat)  =      30.0000  a.u.
     unit-cell volume          =    4200.2820 (a.u.)^3
     number of atoms/cell      =            4
     number of atomic types    =            2
     number of electrons       =        10.00
     number of Kohn-Sham states=           15
     kinetic-energy cutoff     =      40.0000  Ry
     charge density cutoff     =     160.0000  Ry
     Exchange-correlation      =  SLA  PZ   NOGX NOGC ( 1 1 0 0 0)
     EXX-fraction              =        0.00

     celldm(1)=  30.000000  celldm(2)=   1.000000  celldm(3)=   0.155566
     celldm(4)=   0.000000  celldm(5)=   0.000000  celldm(6)=   0.000000

     crystal axes: (cart. coord. in units of alat)
               a(1) = (   1.000000   0.000000   0.000000 )  
               a(2) = (   0.000000   1.000000   0.000000 )  
               a(3) = (   0.000000   0.000000   0.155566 )  

     reciprocal axes: (cart. coord. in units 2 pi/alat)
               b(1) = (  1.000000  0.000000  0.000000 )  
               b(2) = (  0.000000  1.000000  0.000000 )  
               b(3) = (  0.000000  0.000000  6.428140 )  


     PseudoPot. # 1 for C  read from file:
     ../pseudo/C.pz-vbc.UPF
     MD5 check sum: ab53dd623bfeb79c5a7b057bc96eae20
     Pseudo is Norm-conserving, Zval =  4.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  269 points,  1 beta functions with: 
                l(1) =   0

     PseudoPot. # 2 for H  read from file:
     ../pseudo/H.pz-vbc.UPF
     MD5 check sum: 90becb985b714f09656c73597998d266
     Pseudo is Norm-conserving, Zval =  1.0
     Generated by new atomic code, or converted to UPF format
     Using radial grid of  131 points,  0 beta functions with: 

     atomic species   valence    mass     pseudopotential
        C              4.00     1.00000     C ( 1.00)
        H              1.00     1.00000     H ( 1.00)

      2 Sym. Ops. (no inversion) found
          (note:  2 additional sym.ops. were found but ignored
           their fractional translations are incommensurate with FFT grid)


   Cartesian axes

     site n.     atom                  positions (alat units)
         1           C   tau(   1) = (   0.0000000   0.0000000  -0.0368990  )
         2           C   tau(   2) = (   0.0408396   0.0000000   0.0368992  )
         3           H   tau(   3) = (  -0.0678406   0.0000000  -0.0371833  )
         4           H   tau(   4) = (   0.1086802   0.0000000   0.0371835  )

     number of k points=    26
                       cart. coord. in units 2pi/alat
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0400000
        k(    2) = (   0.0000000   0.0000000   0.1285628), wk =   0.0800000
        k(    3) = (   0.0000000   0.0000000   0.2571256), wk =   0.0800000
        k(    4) = (   0.0000000   0.0000000   0.3856884), wk =   0.0800000
        k(    5) = (   0.0000000   0.0000000   0.5142512), wk =   0.0800000
        k(    6) = (   0.0000000   0.0000000   0.6428140), wk =   0.0800000
        k(    7) = (   0.0000000   0.0000000   0.7713768), wk =   0.0800000
        k(    8) = (   0.0000000   0.0000000   0.8999396), wk =   0.0800000
        k(    9) = (   0.0000000   0.0000000   1.0285024), wk =   0.0800000
        k(   10) = (   0.0000000   0.0000000   1.1570652), wk =   0.0800000
        k(   11) = (   0.0000000   0.0000000   1.2856280), wk =   0.0800000
        k(   12) = (   0.0000000   0.0000000   1.4141908), wk =   0.0800000
        k(   13) = (   0.0000000   0.0000000   1.5427536), wk =   0.0800000
        k(   14) = (   0.0000000   0.0000000   1.6713164), wk =   0.0800000
        k(   15) = (   0.0000000   0.0000000   1.7998792), wk =   0.0800000
        k(   16) = (   0.0000000   0.0000000   1.9284419), wk =   0.0800000
        k(   17) = (   0.0000000   0.0000000   2.0570047), wk =   0.0800000
        k(   18) = (   0.0000000   0.0000000   2.1855675), wk =   0.0800000
        k(   19) = (   0.0000000   0.0000000   2.3141303), wk =   0.0800000
        k(   20) = (   0.0000000   0.0000000   2.4426931), wk =   0.0800000
        k(   21) = (   0.0000000   0.0000000   2.5712559), wk =   0.0800000
        k(   22) = (   0.0000000   0.0000000   2.6998187), wk =   0.0800000
        k(   23) = (   0.0000000   0.0000000   2.8283815), wk =   0.0800000
        k(   24) = (   0.0000000   0.0000000   2.9569443), wk =   0.0800000
        k(   25) = (   0.0000000   0.0000000   3.0855071), wk =   0.0800000
        k(   26) = (   0.0000000   0.0000000  -3.2140699), wk =   0.0400000

     Dense  grid:   143691 G-vectors     FFT dimensions: ( 125, 125,  20)

     Largest allocated arrays     est. size (Mb)     dimensions
        Kohn-Sham Wavefunctions         0.69 Mb     (   3007,   15)
        NL pseudopotentials             0.09 Mb     (   3007,    2)
        Each V/rho on FFT grid          0.95 Mb     (  62500)
        Each G-vector array             0.18 Mb     (  23949)
        G-vector shells                 0.05 Mb     (   6184)
     Largest temporary arrays     est. size (Mb)     dimensions
        Auxiliary wavefunctions         2.75 Mb     (   3007,   60)
        Each subspace H/S matrix        0.05 Mb     (  60,  60)
        Each <psi_i|beta_j> matrix      0.00 Mb     (      2,   15)

     The potential is recalculated from file :
     ./SCRATCH/poly_pa.save/charge-density.dat

     Starting wfc are   10 randomized atomic wfcs

     total cpu time spent up to now is        0.4 secs

     per-process dynamical memory:    17.1 Mb

     Band Structure Calculation
     Davidson diagonalization with overlap
     c_bands:  1 eigenvalues not converged

     ethr =  1.00E-10,  avg # of iterations = 54.3

     total cpu time spent up to now is       85.9 secs

     End of band structure calculation

          k = 0.0000 0.0000 0.0000     band energies (ev):

   -20.5452 -13.9796  -8.9286  -8.9224  -7.9802  -0.2266  -0.0902   0.3023
     0.5494   0.6240   0.6866   1.0824   1.2332   1.3969   1.5632

          k = 0.0000 0.0000 0.1286     band energies (ev):

   -20.5385 -13.9731  -8.9460  -8.9140  -7.9903  -0.2166  -0.0817   0.3091
     0.5592   0.6334   0.6961   1.0924   1.2430   1.4065   1.5716

          k = 0.0000 0.0000 0.2571     band energies (ev):

   -20.5189 -13.9546  -9.0034  -8.8884  -8.0205  -0.1873  -0.0561   0.3283
     0.5886   0.6615   0.7245   1.1221   1.2725   1.4351   1.5960

          k = 0.0000 0.0000 0.3857     band energies (ev):

   -20.4859 -13.9239  -9.1031  -8.8444  -8.0620  -0.1386  -0.0136   0.3594
     0.6377   0.7083   0.7719   1.1716   1.3216   1.4828   1.6366

          k = 0.0000 0.0000 0.5143     band energies (ev):

   -20.4394 -13.8806  -9.2473  -8.7821  -8.1070  -0.0709   0.0455   0.4014
     0.7064   0.7735   0.8384   1.2409   1.3904   1.5494   1.6929

          k = 0.0000 0.0000 0.6428     band energies (ev):

   -20.3801 -13.8262  -9.4396  -8.7033  -8.1522   0.0156   0.1205   0.4518
     0.7947   0.8569   0.9242   1.3297   1.4788   1.6346   1.7645

          k = 0.0000 0.0000 0.7714     band energies (ev):

   -20.3082 -13.7606  -9.6772  -8.6079  -8.1945   0.1204   0.2110   0.5076
     0.9026   0.9579   1.0293   1.4382   1.5869   1.7382   1.8507

          k = 0.0000 0.0000 0.8999     band energies (ev):

   -20.2224 -13.6832  -9.9479  -8.4943  -8.2328   0.2434   0.3164   0.5653
     1.0301   1.0762   1.1541   1.5662   1.7146   1.8596   1.9513

          k = 0.0000 0.0000 1.0285     band energies (ev):

   -20.1239 -13.5964 -10.2461  -8.3638  -8.2709   0.3831   0.4360   0.6175
     1.1773   1.2113   1.2986   1.7137   1.8618   1.9976   2.0653

          k = 0.0000 0.0000 1.1571     band energies (ev):

   -20.0132 -13.5020 -10.5683  -8.3135  -8.2176   0.5348   0.5724   0.6551
     1.3439   1.3626   1.4629   1.8806   2.0286   2.1508   2.1925

          k = 0.0000 0.0000 1.2856     band energies (ev):

   -19.8893 -13.4006 -10.9011  -8.3590  -8.0537   0.6689   0.6853   0.7371
     1.5301   1.5301   1.6471   2.0671   2.2148   2.3170   2.3338

          k = 0.0000 0.0000 1.4142     band energies (ev):

   -19.7518 -13.2970 -11.2360  -8.4114  -7.8725   0.6468   0.8324   0.9293
     1.7137   1.7359   1.8511   2.2730   2.4198   2.4911   2.4939

          k = 0.0000 0.0000 1.5428     band energies (ev):

   -19.6021 -13.2006 -11.5634  -8.4740  -7.6742   0.5767   0.9800   1.1422
     1.9136   1.9613   2.0750   2.4985   2.6396   2.6709   2.6815

          k = 0.0000 0.0000 1.6713     band energies (ev):

   -19.4401 -13.1315 -11.8595  -8.5481  -7.4611   0.4499   1.1272   1.3735
     2.1309   2.2061   2.3186   2.7433   2.8510   2.8824   2.8990

          k = 0.0000 0.0000 1.7999     band energies (ev):

   -19.2657 -13.1331 -12.0777  -8.6350  -7.2315   0.2655   1.2732   1.6228
     2.3666   2.4705   2.5819   3.0075   3.0703   3.1013   3.1617

          k = 0.0000 0.0000 1.9284     band energies (ev):

   -19.0786 -13.2591 -12.1599  -8.7344  -6.9864   0.0279   1.4184   1.8900
     2.6224   2.7543   2.8647   3.2909   3.3125   3.3422   3.4457

          k = 0.0000 0.0000 2.0570     band energies (ev):

   -18.8802 -13.5026 -12.1150  -8.8474  -6.7262  -0.2555   1.5652   2.1744
     2.8994   3.0576   3.1671   3.5796   3.5918   3.6091   3.7497

          k = 0.0000 0.0000 2.1856     band energies (ev):

   -18.6698 -13.8091 -11.9972  -8.9768  -6.4514  -0.5760   1.7170   2.4759
     3.1983   3.3802   3.4888   3.8715   3.8932   3.9196   4.0733

          k = 0.0000 0.0000 2.3141     band energies (ev):

   -18.4489 -14.1389 -11.8421  -9.1199  -6.1629  -0.9223   1.8782   2.7941
     3.5194   3.7222   3.8297   4.1877   4.2081   4.2604   4.4165

          k = 0.0000 0.0000 2.4427     band energies (ev):

   -18.2187 -14.4759 -11.6669  -9.2774  -5.8623  -1.2889   2.0530   3.1283
     3.8627   4.0834   4.1895   4.5275   4.5463   4.6209   4.7792

          k = 0.0000 0.0000 2.5713     band energies (ev):

   -17.9776 -14.8117 -11.4771  -9.4455  -5.5488  -1.6636   2.2447   3.4784
     4.2274   4.4638   4.5677   4.8902   4.9056   4.9974   5.1613

          k = 0.0000 0.0000 2.6998     band energies (ev):

   -17.7294 -15.1419 -11.2796  -9.6254  -5.2278  -2.0412   2.4559   3.8417
     4.6113   4.8632   4.9332   4.9640   5.2750   5.2975   5.4071

          k = 0.0000 0.0000 2.8284     band energies (ev):

   -17.4786 -15.4574 -11.0768  -9.8130  -4.9036  -2.4146   2.6880   4.2063
     4.5823   5.0255   5.2811   5.3729   5.6809   5.7009   5.8229

          k = 0.0000 0.0000 2.9569     band energies (ev):

   -17.2323 -15.7513 -10.8728 -10.0066  -4.5876  -2.7673   2.9406   4.1886
     4.6420   5.4507   5.7171   5.7735   6.1066   6.1266   6.2577

          k = 0.0000 0.0000 3.0855     band energies (ev):

   -17.0205 -15.9917 -10.6772 -10.1960  -4.3141  -3.0639   3.2092   3.8697
     5.0321   5.8004   5.9082   6.1696   6.3425   6.5457   6.5725

          k = 0.0000 0.0000-3.2141     band energies (ev):

   -16.9242 -16.0985 -10.5557 -10.3183  -4.1871  -3.1998   3.4153   3.6442
     5.4006   5.4701   6.3355   6.3915   6.6199   6.6344   6.7493

     Writing output data file poly_pa.save
 
     init_run     :      0.16s CPU      0.17s WALL (       1 calls)
     electrons    :     85.08s CPU     85.56s WALL (       1 calls)

     Called by init_run:
     wfcinit      :      0.01s CPU      0.00s WALL (       1 calls)
     potinit      :      0.04s CPU      0.04s WALL (       1 calls)

     Called by electrons:
     c_bands      :     85.08s CPU     85.56s WALL (       1 calls)
     v_of_rho     :      0.01s CPU      0.01s WALL (       1 calls)

     Called by c_bands:
     init_us_2    :      0.01s CPU      0.01s WALL (      26 calls)
     cegterg      :     82.38s CPU     82.56s WALL (      83 calls)

     Called by *egterg:
     h_psi        :     72.07s CPU     72.23s WALL (    1521 calls)
     g_psi        :      0.79s CPU      0.82s WALL (    1412 calls)
     cdiaghg      :      1.60s CPU      1.74s WALL (    1438 calls)

     Called by h_psi:
     add_vuspsi   :      0.22s CPU      0.26s WALL (    1521 calls)

     General routines
     calbec       :      0.45s CPU      0.35s WALL (    1521 calls)
     fft          :      0.00s CPU      0.02s WALL (       3 calls)
     fftw         :     65.38s CPU     65.86s WALL (   22350 calls)
     davcio       :      0.01s CPU      0.04s WALL (      52 calls)
 
     Parallel routines
     fft_scatter  :     16.10s CPU     15.94s WALL (   22353 calls)
 
     PWSCF        :  1m26.36s CPU     1m27.09s WALL

 
   This run was terminated on:  10:22: 6  28Feb2013            

=------------------------------------------------------------------------------=
   JOB DONE.
=------------------------------------------------------------------------------=
